src/plugins/lightscript.js

```diff
@@ -197,7 +197,8 @@ pp.parseWhiteBlock = function (allowDirectives?, isIfExpression?) {
   // never parse directives if curly braces aren't used (TODO: document)
   this.parseBlockBody(node, false, false, indentLevel);
   this.addExtra(node, "curly", false);
-  if (!node.body.length) this.unexpected(node.start, "Expected an Indent or Statement");
+  if (!isArrowBody && !node.body.length)
+    this.unexpected(node.start, "Expected an Indent or Statement");
```

Comment:
nit: same line or curly please (surprised this isn't in the eslint rules... hmm)